From da89cf5dd7538dddb3c4f001d25e1378d6f1f663 Mon Sep 17 00:00:00 2001
From: robert_sun <robert_sun@realsil.com.cn>
Date: Wed, 24 Apr 2019 19:28:08 +0800
Subject: [PATCH] remove serivce authorization for a2dp

---
 profiles/audio/avdtp.c | 17 +++++++++--------
 1 file changed, 9 insertions(+), 8 deletions(-)

diff --git a/profiles/audio/avdtp.c b/profiles/audio/avdtp.c
index ba5f0e581..2352cb70c 100644
--- a/profiles/audio/avdtp.c
+++ b/profiles/audio/avdtp.c
@@ -2506,14 +2506,15 @@ static void avdtp_confirm_cb(GIOChannel *chan, gpointer data)
 	session->io_id = g_io_add_watch(chan, G_IO_ERR | G_IO_HUP | G_IO_NVAL,
 					(GIOFunc) session_cb, session);
 
-	session->auth_id = btd_request_authorization(&src, &dst,
-							ADVANCED_AUDIO_UUID,
-							auth_cb, session);
-	if (session->auth_id == 0) {
-		avdtp_set_state(session, AVDTP_SESSION_STATE_DISCONNECTED);
-		goto drop;
-	}
-
+	//session->auth_id = btd_request_authorization(&src, &dst,
+	//						ADVANCED_AUDIO_UUID,
+	//						auth_cb, session);
+	//if (session->auth_id == 0) {
+	//	avdtp_set_state(session, AVDTP_SESSION_STATE_DISCONNECTED);
+	//	goto drop;
+	//}
+
+	auth_cb(NULL, session);
 	return;
 
 drop:
-- 
2.21.0

