diff --git a/net/bluetooth/l2cap_core.c b/net/bluetooth/l2cap_core.c
index 2a7fb517d..833bb9969 100644
--- a/net/bluetooth/l2cap_core.c
+++ b/net/bluetooth/l2cap_core.c
@@ -1507,8 +1507,8 @@ static void l2cap_le_conn_ready(struct l2cap_conn *conn)
 	     hcon->le_conn_interval > hcon->le_conn_max_interval)) {
 		struct l2cap_conn_param_update_req req;
 
-		req.min = cpu_to_le16(hcon->le_conn_min_interval);
-		req.max = cpu_to_le16(hcon->le_conn_max_interval);
+		req.min = 0x0006;
+		req.max = 0x0006;
 		req.latency = cpu_to_le16(hcon->le_conn_latency);
 		req.to_multiplier = cpu_to_le16(hcon->le_supv_timeout);
